src/to-have-class.js

```diff
@@ -1,6 +1,20 @@
 import {matcherHint, printExpected} from 'jest-matcher-utils'
 import {checkHtmlElement, getMessage} from './utils'
 
+function getExpectedClassNamesAndOptions(params) {
+  const lastParam = params.pop()
+  let expectedClassNames, options
+
+  if (typeof lastParam === 'object') {
+    expectedClassNames = params
+    options = lastParam
+  } else {
+    expectedClassNames = params.concat(lastParam)
+    options = {}
```

Comment:
Nit pick: should be `{ exact: false }`.

```suggestion
    options = { exact: false }
```